03b_model/src/run_model.py

```diff
@@ -54,12 +99,41 @@ def select_inputs_targets(inputs, target, train_start_date, test_end_date, out_d
     #inputs_df = inputs_df.join(nerrs_input)
     inputs_df = inputs_df[train_start_date:test_end_date]
     
+    if inc_ante:
+        
+        with open("03b_model/model_config.yaml", 'r') as stream:
+            config = yaml.safe_load(stream)
```

Comment:
We should be able to delete this because we read in the config as a global variable in line 23
```suggestion
```